instances/atlas/serviceLink/SWMA_2018/SWMA_2018_lh_SM-Ins_0_MPM.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/serviceLink/SWMA_2018_lh_SM-Ins_0_MPM",
+    "@type": "https://openminds.ebrains.eu/core/ServiceLink",
+    "dataLocation": {
+        "@id": "https://openminds.ebrains.eu/instances/parcellationEntityVersion/SWMA_2018_lh_SM-Ins_0"
+    },
+    "name": "probability map",
```

Comment:
ServiceLinks for *_MPM.jsonld should have `"name": "maximum probability map"`

Reply:
I think this needs to be corrected for all `*_MPM.jsonld`

Reply:
can't believe I missed this. 

It should be fixed now